roles/serviceassurance/tasks/component_alertmanager.yml

```diff
@@ -0,0 +1,35 @@
+- name: Set default Alertmanager manifest
+  set_fact:
+    alertmanager_manifest: |
+      apiVersion: monitoring.coreos.com/v1
+      kind: Alertmanager
+      metadata:
+        labels:
+          alertmanager: '{{ meta.name }}'
+        name: '{{ meta.name }}'
+        namespace: '{{ meta.namespace }}'
+      spec:
+        replicas: 1
```

Comment:
I've added a card (#861) to our triage backlog about HA Alertmanager. It may be as simple as putting a 2 in this field, but there may also be adjustments needed on the prometheus side, or a proper way to do it via the operator.

https://prometheus.io/docs/alerting/alertmanager/#high-availability

Reply:
From what I've been reading if you just set the replicas to > 1 all the Alertmanagers should coordinate with each other to notice who has sent a notification. When configuring Prometheus, alerts go to all available alertmanagers. The coordination among the AMs is what results in only a single fired event.

Looks like the default docs around using Prometheus Operator suggests using 3 AMs. The work will be around the testing, but the configuration change likely just follows the same pattern as ElasticSearch, where the `main.yml` will get a `set_fact` with `alertmanager_replicas: 3` when `high_available_enabled` is `true`. I think that'll likely be the only change required.